.eslintrc.json

```diff
@@ -1,3 +1,14 @@
 {
-  "extends": "./node_modules/gts/"
+  "extends": [ 
+    "./node_modules/gts/",
+    "next/core-web-vitals"
+  ],
+  "overrides": [
+    {
+      "files": ["test/**/*.ts", "*.config.ts"],
+      "rules": {
+        "node/no-unpublished-import": ["off"]
```

Comment:
This rule should only be present for our main code. Test and config code can safely rely on unpublished packages (published means in our `depedencies` part of our `package.json`